src/ucp/tag/tag_send.c

```diff
@@ -284,31 +315,27 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_sync_nb,
     ucs_trace_req("send_sync_nb buffer %p count %zu tag %"PRIx64" to %s cb %p",
                   buffer, count, tag, ucp_ep_peer_name(ep), cb);
 
-    if (ucp_ep_config(ep)->key.err_mode == UCP_ERR_HANDLING_MODE_PEER) {
-        ret = UCS_STATUS_PTR(UCS_ERR_UNSUPPORTED);
-        goto out;
-    }
+    status = ucp_tag_send_common(ep, buffer, count, tag, &param, UCP_REQUEST_FLAG_SYNC);
```

Comment:
can we move UCP_REQUEST_FLAG_SYNC to be inside attr_mask ?
user could specify if he wants blcoking behavior or not..